### PR TITLE
Bug/ab2d 1667 earliest data date is jan 1 2020

### PR DESCRIPTION
**Pull request by lailasharshar**
### Summary

Previously, we only checked to see if the data was past the attestation date but if they attested in December of last year, they still can only have data past 1/1/2020. Fixed that. Also noticed a bug in eventlogger and fixed it.

### Checklist

- [ ] Tests and linting pass
- [ ] Code checked for PHI/PII exposure

### Security
What secure items does this touch, how does this PR affect our security posture, etc?

### CODE REVIEW

File: eventlogger/src/test/java/gov/cms/ab2d/eventlogger/eventloggers/sql/AllMapperEventTest.java [hunk 1]
```diff
@@ -48,11 +48,27 @@ void exceptionApiRequestTests() {
                 new ApiRequestEventMapper(null).log(new ErrorEvent()));
     }
 
+    @Test
+    void createError() {
+        // There is no mapping. Make sure it doesn't create an issue
+        BeneficiarySearchEvent bene = new BeneficiarySearchEvent();
+        sqlEventLogger.log(bene);
+        sqlEventLogger.updateAwsId("TEST", bene);
```
Comment (adaykin) [inline eventlogger/src/test/java/gov/cms/ab2d/eventlogger/eventloggers/sql/AllMapperEventTest.java:56]: Is the idea with this test and the one below to see if it doesn't throw an exception?

File: .travis.yml [hunk 2]
```diff
@@ -40,6 +40,7 @@ before_install:
 - export AB2D_HICN_HASH_PEPPER=`cat bfd_sbx_hash_pepper`
 - export AB2D_HICN_HASH_ITER=1000
 - export AB2D_CLAIMS_SKIP_BILLABLE_PERIOD_CHECK=true
+- export AB2D_EARLIEST_DATA_DATE=01/01/1900
```
Comment (adaykin) [inline .travis.yml:43]: It's 1900 here and 2020 elsewhere is that b/c our tests are relying on earlier dates?

File: eventlogger/src/test/java/gov/cms/ab2d/eventlogger/eventloggers/sql/AllMapperEventTest.java (hunk 1 above)
Comment (lailasharshar) [reply eventlogger/src/test/java/gov/cms/ab2d/eventlogger/eventloggers/sql/AllMapperEventTest.java:56]: Yes, even though it doesn't have an sql mapping. I don't want logging to mess anything up.

File: .travis.yml (hunk 2 above)
Comment (lailasharshar) [reply .travis.yml:43]: Correct. The real one is 2020, but the e2e tests rely on test data from 199-2002.

**Pull request merged**